extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -97,26 +95,22 @@ public function onEditPageMakeGlobalVariablesScript( array &$aVars ) {
 	 * @return bool
 	 */
 	public function onEditPageShowEditFormFields( \EditPage $editPage, \OutputPage $out ) {
-		global $wgCityId, $wgEnablePortableInfoboxBuilderExt;
+		global $wgCityId;
 
 		$context = \RequestContext::getMain();
 		$title = $context->getTitle();
 
 		if ( ( new Permissions() )->shouldDisplayEntryPoint( $context->getUser(), $title ) ) {
 			$types = $this->getTemplateTypeForEdit( $editPage->getTitle(), $wgCityId );
 
-			$out->addHTML( \Html::hidden( 'templateClassificationTypeCurrent', $types['current'],
-				['autocomplete' => 'off'] ) );
-			$out->addHTML( \Html::hidden( 'templateClassificationTypeNew', $types['new'],
-				['autocomplete' => 'off'] ) );
+			$out->addHTML( \Html::hidden( 'templateClassificationTypeCurrent', $types[ 'current' ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "current" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)